packages/math-input/src/components/keypad/button.stories.tsx

```diff
@@ -1,29 +1,75 @@
 import {action} from "@storybook/addon-actions";
-import {withKnobs} from "@storybook/addon-knobs";
+import {ComponentStory} from "@storybook/react";
 import * as React from "react";
 
-import KeypadButton from "./button";
+import KeyConfigs from "../../data/key-configs";
+import Keys from "../../data/keys";
+
+import {KeypadButton, KeypadButtonProps} from "./keypad-page-items";
 
 export default {
     title: "Keypad Button",
-    decorators: [withKnobs],
-    parameters: {
-        backgrounds: {
-            values: [
-                {name: "dark background", value: "#DBDCDD", default: true},
-            ],
+    argTypes: {
+        keyConfig: {
+            control: "select",
+            options: {...KeyConfigs},
+            defaultValue: KeyConfigs[Keys.PLUS],
+        },
+        tintColor: {
```

Comment:
Did you say this isn't doing anything? Should it be removed then?

Reply:
I think I was wrong. Maybe `aphrodite` is doing something wild here? It looks like it's working when we render a `SecondaryKeypadButton` and an `ActionKeypadButton` (where the only difference is this prop is hard-coded), but when we render a `KeypadButton` the background color css gets `!important` applied and we can't seem to change it here. I can't find where we are applying that in the code. Something's wacky about this prop and it needs a little more investigation.